lib/fennec/udp.ex

```diff
@@ -2,8 +2,13 @@ defmodule Fennec.UDP do
   @moduledoc """
   UDP STUN server
 
-  The easiest way to start a server is to hook it up
-  to your supervision tree:
+  The easiest way to start a server is to start it under Fennec's
+  supervision tree using configuration:
```

Comment:
...is to spawn it under Fennec's supervision tree providing the following configuration...